prometheus_hardware_exporter/__main__.py

```diff
@@ -42,13 +35,12 @@ def main() -> None:
     root_logger.setLevel(logging.getLevelName(config.level))
 
     exporter = Exporter(config.port)
-    exporter.register(MegaRAIDCollector())
-    exporter.register(IpmiDcmiCollector())
-    exporter.register(IpmiSensorsCollector())
-    exporter.register(IpmiSelCollector())
-    exporter.register(LSISASControllerCollector(2))
-    exporter.register(LSISASControllerCollector(3))
-    exporter.register(PowerEdgeRAIDCollector())
+
+    collector_filter = set(config.disable_collector)
+    for name, collector in collector_registries.items():
+        if name.lower() in collector_filter:
```

Comment:
I think it would be nice to have some log message here. e.g. `logger.info("collector %s is disabled", name)`

Reply:
+1

Reply:
DONE

Reply:
This is not good practice to use root logger to making logs, we should use root_logger only for configuration. At least I think that's good practice. What do you think?

Reply:
I am not sure if this is what you mean: are you suggesting that 

```
logger = getLogger(__main__)
root_logger = getLogger() # keeping this for setLevel
...
for name, collector in collector_registries.items():
   if name.lower() in collector_filter:
       logger.info(....)
```
?

Reply:
Initially that was my idea, but after checking it we should do.
```python
logger = getLogger(__main__)
... 
for name, collector in collector_registries.items():
   if name.lower() in collector_filter:
       logger.info(....)

def main():
...
    root_logger = logging.getLogger()
    root_logger.setLevel(...)
```

Reply:
I've updated this part.

Reply:
Thanks :+1: